subsystems/elevatorsubsystem.py

```diff
@@ -126,21 +141,31 @@ def getElevatorPosition(self) -> float:
 
     def setL4Position(self) -> None:
         self.state = self.ElevatorState.L4Position
+        self.elevatorManualModePublisher.set(False)
```

Comment:
Shoul these technically be 1 line up so the L4 doesn't overwrite it? all happens in 1 robot tick anyway so maybe it doesn't matter idk